confluent_kafka/src/Producer.c

```diff
@@ -251,7 +293,8 @@ static PyObject *Producer_produce (Handle *self, PyObject *args,
 	int value_len = 0, key_len = 0;
 	int partition = RD_KAFKA_PARTITION_UA;
 	PyObject *dr_cb = NULL, *dr_cb2 = NULL, *partitioner_cb = NULL;
-	rd_kafka_topic_t *rkt;
+        long long timestamp = 0;
```

Comment:
re indent: I'm trying to move away from tabs to spaces.
I should probably just do a forklift update to untabify everything in one commit.